packages/flutter_tools/lib/src/build_system/targets/macos.dart

```diff
@@ -247,6 +266,29 @@ class DebugBundleFlutterAssets extends Target {
     } catch (err) {
       throw Exception('Failed to copy precompiled runtimes: $err');
     }
+    // Create symlink to current version.
+    try {
+      final Link currentVersion = outputDirectory.parent
+          .childLink('Current');
+      if (!currentVersion.existsSync()) {
+        currentVersion.createSync(outputDirectory.path);
+      }
+      // Create symlink to current resources
+      final Link currentResources = frameworkRootDirectory
+          .childLink('Resources');
+      if (!currentResources.existsSync()) {
+        currentResources.createSync(fs.path.join(currentVersion.path, 'Resources'));
+      }
+      // Create symlink to current resources
+      final Link currentFramework = frameworkRootDirectory
+          .childLink('App');
+      if (!currentFramework.existsSync()) {
+        currentFramework.createSync(fs.path.join(currentVersion.path, 'App'));
+      }
+    } on FileSystemException {
+      throw Exception('Failed to create symlinks for framework. try removing '
+        'the "macos/Fluter/ephemeral" directory and rerunning');
```

Comment:
Flutter

(Or even better, get this from the project instead of hard-coding the string, so it won't get stale.)

Reply:
Done